devel/fortio/cmd/fortio/pingsrv.go

```diff
@@ -146,17 +145,16 @@ func healthCheck(serverAddr string, svcname string, n int) {
 	fmt.Printf("Statuses %v\n", statuses)
 }
 
-func main() {
-	flag.Parse()
-	if *hostFlag != "" {
-		// TODO doesn't work for ipv6 addrs etc
-		dest := fmt.Sprintf("%s:%d", *hostFlag, *portFlag)
-		if *doHealthFlag {
-			healthCheck(dest, *healthSvcFlag, *countFlag)
-		} else {
-			clientCall(dest, *countFlag, *payloadFlag)
-		}
+func grpcClient() {
+	if len(flag.Args()) != 1 {
```

Comment:
I am not sure I understand. (in addition to minimizing the diff from when this was a standalone main) the reason the code is here is because not all sub commands in the switch need an argument, and the error message is specific to the grpcClient subcommand

where is it directly operating on the args ?